pkg/common/parameters.go

```diff
@@ -155,6 +158,12 @@ func ExtractAndDefaultParameters(parameters map[string]string, driverName string
 				return p, fmt.Errorf("parameters contain invalid provisionedThroughputOnCreate parameter: %w", err)
 			}
 			p.ProvisionedThroughputOnCreate = paramProvisionedThroughputOnCreate
+		case ParameterForceAttach:
```

Comment:
do you want to validate that it's only set for repd?

Reply:
Maybe not here (especially if we go with the availabilityClass idea), but certainly should validate before making the attach call, as using force attach on a zonal disk results in an error according to the docs.

Reply:
Done, I ended up doing the validation in the create volume request.